INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
Ah, yes, the solution I wanted to point to was the second one, not the ugly `chown`. I _believe_ the official NPM docs discourage using `sudo` in that situation, so I don't think we should recommend that in our installation instructions.